src/plugins/assert/assertHttpRegionParser.ts

```diff
@@ -69,8 +69,8 @@ export async function parseAssertLine(
         nextParserLine: next.value.line,
         symbols: [
           {
-            name: match.groups.key,
-            description: match.groups.value,
+            name: match.groups.key ?? "script",
```

Comment:
match.groups.key seems to be a Copy Paste error of me. match.groups.type should be correct and is always defined
```suggestion
            name: `assert ${match.groups.key}`,
```